packages/app/src/specs/InlineSpecList.cy.tsx

```diff
@@ -49,11 +49,16 @@ describe('InlineSpecList', () => {
 
     it('should support fuzzy sort', () => {
       mountInlineSpecList()
-      cy.get('input').type('scome', { force: true })
+      cy.get('input').type('compspec', { force: true })
 
       cy.get('li').should('have.length', 4)
       .should('contain', 'src/components')
       .and('contain', 'Spec-A.spec.tsx')
+
+      // Don't want the search to be too fuzzy
+      // sr => src, co => components, sp => spec
```

Comment:
comment doesn't seem to quite line up with what's being typed - "sp" vs "se"

Reply:
Addressed in [06a2c94](https://github.com/cypress-io/cypress/pull/23325/commits/06a2c946f798336fdd3672c7b124c8ceaa872909)